### ISSUE TITLE
potentially spurious spaces in template definitions

### ISSUE THREAD

**Issue by u-fischer** [josephwright/beamer#425]
I just came across this template definition (in `beameroutherthemeinfolines.sty):

```
\defbeamertemplate*{footline}{infolines theme}
{    %<--- spurious space here
  \leavevmode%
  \hbox{%
....
```

When used as is the space doesn't matter. But if one tries to add something at the begin of the template wit `\addbeamertemplate`  one gets mysterious gaps. See e.g. this answer:
https://tex.stackexchange.com/a/401747/2388.

**Comment by louisstuart96** [comment]
We're planning to find all the 'extra spaces' like this. We would be appreciate if you can help us find more. You can send a pull request to correct what you have found.

**Comment by josephwright** [comment]
Closed by f0576fc64bb16730561cf59a77f4d312dc65e33c.
